src/controller/breakingChanges/field.ts

```diff
@@ -14,6 +14,7 @@ export class FieldChange implements BreakingChangeService {
 	private types = [
 		ChangeType.FieldRemoved,
 		ChangeType.FieldTypeChanged,
+		ChangeType.FieldArgumentAdded,
```

Comment:
If the field argument is added and is optional can be that is not a breaking change, have you checked what the library from Apollo says ?

Reply:
Yes. Its only considered as a breakingChange if you add a required argument. If its an optional field its considered as Dangerous = "DANGEROUS"